code/modules/mob/living/carbon/human/species_types/synths.dm

```diff
@@ -27,10 +26,13 @@
 	id = "military_synth"
 	armor = 25
 	punchdamage = 14
-	disguise_fail_health = 50
+	inherent_traits = list(TRAIT_NODISMEMBER,TRAIT_NOLIMBDISABLE,TRAIT_NOHUNGER,TRAIT_NOBREATH, TRAIT_NOHUNGER, TRAIT_TOXIMMUNE, TRAIT_DOOR_PRYER, TRAIT_NOSTAMCRIT, TRAIT_STRONG_GRABBER)
```

Comment:
TRAIT_DOOR_PRYER is an item trait not a mob trait.

Reply:
pfffft whoops, was thinking of Goon